.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve Juju
+labels:
+  - kind/bug
+  - needs-triage
+body:
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: "Please describe the bug for us: "
+      placeholder: ex. "`juju status` returns the error `not found`"
+    validations:
+      required: true
+
+  - type: input
+    id: Version
+    attributes:
+      label: Juju version
+      description: "What version of Juju are you using?"
+      placeholder: ex. 3.6.1
+    validations:
+      required: true
+
+  - type: input
+    id: Cloud
+    attributes:
+      label: Cloud
+      description: What cloud are you using?
+      placeholder: ex. AWS
+    validations:
+      required: false
```

Comment:
same here for allowed cloud names
or can we make this a drop down?

Reply:
A drop-box would be great